bindings/ruby/test/test_plugin_ModelTask.rb

```diff
@@ -134,17 +134,6 @@
             assert (4..6).include?(samples.size)
         end
 
-        it 'fails if the joint is given relative to the model' do
```

Comment:
Replace it by a "it handles joints given relatively to the model" test. This will ensure that someone has to change tests to re-establish the old behavior.